tests/test_warm_reboot.py

```diff
@@ -836,6 +835,10 @@ def test_swss_port_state_syncup(dvs, testlog):
     dvs.stop_swss()
     time.sleep(3)
 
+    dvs.runcmd("ip addr flush dev Ethernet0")
```

Comment:
Can we use the same method as deleting from Interface table? `intf_tbl._del`.
 It looks like we add one way and flush another way